content/src/fieldConfig/business-formation.json

```diff
@@ -153,7 +154,10 @@
         "tableHeader": "Name,Address,Sign*,Actions",
         "titleLabel": "Title",
         "description": "Authorized Representative(s) should sign below by adding their full name. You can have up to 10 signers listed.",
-        "errorInlineAdditionalSignerName": "Enter the first and last name of each signer."
+        "errorInlineAdditionalSignerName": "Enter the first and last name of each signer.",
+        "errorInLineNameAndSignatureMissing": "Enter a signer’s name and click the checkbox to authorize their signature.",
```

Comment:
[pebble] Can we update this to `errorInlineSignerNameAndCheckbox`? I don't think we need `Missing` since the `error` prefix denotes something is missing or incorrect? Also, this naming convention holds to the `Inline` and `SignerName` patterns that already exist?